service/history/workflow/mutable_state_impl.go

```diff
@@ -474,7 +474,6 @@ func (ms *MutableStateImpl) getCurrentBranchTokenAndEventVersion(eventID int64)
 
 // SetHistoryTree set treeID/historyBranches
 func (ms *MutableStateImpl) SetHistoryTree(
-	ctx context.Context,
```

Comment:
This is the change.

Reply:
@norberthu Do you think it's possible that in the future NewHistoryBranch() persistence call may need a context?

Reply:
@yycptt after our latest discussion from about a week ago, I believe the chances of needing this context while new'ing a history branch is near zero. Please go ahead and remove it. Thanks!